### ISSUE TITLE
[GUIDE] read me first! How can I help?

### ISSUE THREAD

**Issue by cskiwi** [Badminton-Apps/core#9]
Hello there,

Glad you are showing interest in working on our project on the competition app. This info text is created to get you started on resolving issues and taking part in the project in the github environment.

If you haven't got the project running on your local machine, we suggest you read the [readme](https://github.com/Badminton-Apps/core/blob/main/README.md#development-setup) page and [contributing](https://github.com/Badminton-Apps/core/blob/main/CONTRIBUTING.md) page.

For first time contributors (to the project), filter on the label: ["good first issue"](https://github.com/Badminton-Apps/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). These are issues on which you can easily start working without an experienced knowledge on app development or programming. Don't hesitate to try these issues out!

Next, try to first resolve the ["required"](https://github.com/Badminton-Apps/core/issues?q=is%3Aopen+is%3Aissue+label%3Arequired) labels before starting on any ["optional"](https://github.com/Badminton-Apps/core/labels/optional) issues, as the "required" labels are needed for the next [milestone](https://github.com/Badminton-Apps/core/milestones?direction=asc&sort=due_date&state=open). These required issues constitute the basic functionalities of the badminton app and are essential to launch the app to the greater badminton community.

Feel free to launch multiple Pull requests per issue, the more we try as contributors, the better the final version will get ;)

Also, feel free to add any new issues if you have an interesting functionality in mind or if you ought a programming challenge to be significant in which we need to put some efforts and which is best tackled in a separate issue.

Finally, if you have any questions or ideas for some code under construction or to us as project leaders (Glenn Latomme and Badminton Vlaanderen), don't hesitate to use the comment sections.

And to end, of course, we wish you good luck and lots of fun by working on this project.

Together we can do it!
